src/sentry/static/sentry/app/views/organizationIntegrations/integrationPluginRow.tsx

```diff
@@ -0,0 +1,111 @@
+import {withTheme} from 'emotion-theming';
+import PropTypes from 'prop-types';
+import React from 'react';
+import styled from '@emotion/styled';
+import Link from 'app/components/links/link';
+import {PanelItem} from 'app/components/panels';
+import {t} from 'app/locale';
+import CircleIndicator from 'app/components/circleIndicator';
+import PluginIcon from 'app/plugins/components/pluginIcon';
+import SentryTypes from 'app/sentryTypes';
+import space from 'app/styles/space';
+import {PluginWithProjectList} from 'app/types';
+
+type Props = {
+  plugin: PluginWithProjectList;
+  legacy: boolean;
+};
+
+export default class PluginRow extends React.Component<Props> {
+  static propTypes = {
```

Comment:
@NisanthanNanthakumar no need to include `propTypes` since this component is only used in TS components.